guide/tests/message_transformation_test.bal

```diff
@@ -0,0 +1,42 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/mime;
+import ballerina/io;
+import ballerina/mysql;
+import ballerina/test;
+
+boolean serviceStarted;
+
+function startService() {
+    serviceStarted = test:startServices("message_transformation");
+}
+
+@test:Config {
+    before: "startService",
+    after: "stopService"
+}
+
+function message_transformation_check() {
+    // Invoking the main function
+    endpoint http:Client httpEndpoint { url: "http://localhost:9090" };
+    // Chck whether the server is started
+    test:assertTrue(serviceStarted, msg = "Unable to start the service");
+    json payload = {"id" : 105, "name" : "saneth", "city" : "Colombo 03", "gender" : "male"};
+    json response1 = {"id":105,"city":"Colombo 03","gender":"male","fname":"saneth","results":{"Com_Maths":"A","Physics":"B","Chemistry":"C"}};
```

Comment:
Reformat the code to have 120 line width

Reply:
Done